### PR TITLE
Validate pipeline numeric options to ensure positive integers.

### LINKED ISSUE #4449: Need to reject --pipeline-batch-size 0

### PR DESCRIPTION
**Pull request by andrewvc**
There were no specs here before, and IMHO they're a bit overkill for
this section of code, so I'm not changing that.

Fixes #4449

### CODE REVIEW

File: logstash-core/lib/logstash/runner.rb [hunk 1]
```diff
@@ -79,6 +79,15 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.runner.flag.node_name"),
     :attribute_name => :node_name
 
+  def validate_positive_integer(str_arg)
+    int_arg = str_arg.to_i
+    if str_arg !~ /^\d+$/ || int_arg < 1
```
Comment (suyograo) [inline logstash-core/lib/logstash/runner.rb]: shouldn't this be `int_arg <= 0`?

File: logstash-core/lib/logstash/runner.rb [hunk 2]
```diff
@@ -79,6 +79,15 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.runner.flag.node_name"),
     :attribute_name => :node_name
 
+  def validate_positive_integer(str_arg)
```
Comment (suyograo) [inline logstash-core/lib/logstash/runner.rb]: IMO we should move this to `utils` dir, but we can do that in a separate PR.

File: logstash-core/lib/logstash/runner.rb (hunk 1 above)
Comment (andrewvc) [inline logstash-core/lib/logstash/runner.rb]: Since its an integer `int_arg < 1 == int_arg <= 0`.

File: logstash-core/lib/logstash/runner.rb (hunk 2 above)
Comment (andrewvc) [inline logstash-core/lib/logstash/runner.rb]: I think you're mostly right, but its a little premature. I'd agree if we had two validations, but since its only one, I'd rather not create a one method utility class.

File: logstash-core/lib/logstash/runner.rb (hunk 1 above)
Comment (suyograo) [inline logstash-core/lib/logstash/runner.rb]: oh man, i don't know what i was thinking there (facepalm) I swear i read it as `int_arg < 0`

**Comment by suyograo** [comment]
LGTM
